### PR TITLE
Add example on the use of dcterms:conformsTo with dcat:CatalogRecord

### LINKED ISSUE #1489: Add example on the use of dcterms:conformsTo with a catalog record
See resolution in https://www.w3.org/2022/03/22-dxwgdcat-minutes#r02

The proposal is to add an example in [§14.2.1 Conformance to a standard](https://w3c.github.io/dxwg/dcat/#quality-conformance-statement) to show how to use `dcterms:conformsTo` with a `dcat:CatalogRecord`.

The proposal is implemented via PR https://github.com/w3c/dxwg/pull/1488

### PR DESCRIPTION
**Pull request by andrea-perego**
Summary of changes:
- Added example and introductory text at the end of [§14.2.1 Conformance to a standard](https://raw.githack.com/w3c/dxwg/dcat-ex-record-schema/dcat/index.html#quality-conformance-statement) to show how to use `dcterms:conformsTo` with a `dcat:CatalogRecord` - as per resolution in https://www.w3.org/2022/03/22-dxwgdcat-minutes#r02
- Added Turtle, RDF/XML, and JSON-LD versions of the example in the `dcat/examples/` folder
- Added cross-reference to [§14.2.1 Conformance to a standard](https://raw.githack.com/w3c/dxwg/dcat-ex-record-schema/dcat/index.html#quality-conformance-statement) for guidance on the use `dcterms:conformsTo`
- Replaced "item" with "resource" when referring to an instance of `dcat:Resource` - see https://github.com/w3c/dxwg/issues/1490
- Replaced "item" with "record" when referring to an instance of `dcat:CatalogRecord` - https://github.com/w3c/dxwg/issues/1490
- Updated changelog
- Run US spellchecker
- Editorial fixes

Preview: https://raw.githack.com/w3c/dxwg/dcat-ex-record-schema/dcat/index.html#ex-catalog-record-schema

Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fdxwg%2Fdcat%2F&doc2=https%3A%2F%2Fraw.githack.com%2Fw3c%2Fdxwg%2Fdcat-ex-record-schema%2Fdcat%2Findex.html

### CODE REVIEW

File: dcat/index.html
Comment (riccardoAlbertoni) [inline dcat/index.html]: I would remove teh duplication of .org in the dcterms:conformsTo  uri 

```suggestion
  dcterms:conformsTo &lt;https://www.w3.org/TR/vocab-dcat/&gt; ;
```

**Review by riccardoAlbertoni — APPROVED** [review]

**Comment by riccardoAlbertoni** [comment]
We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.

**Comment by andrea-perego** [comment]
@riccardoAlbertoni said:

> We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.

Agreed. Let's keep this on hold until #1486 is merged.

**Comment by andrea-perego** [comment]
> @riccardoAlbertoni said:
> 
> > We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.
> 
> Agreed. Let's keep this on hold until #1486 is merged.

I updated the changelog, and I also took the opportunity to fix some typos and editorial issues. I also run the US spellchecker.

Some of the typos concern two property definitions not in the DCAT namespace. So the fixes should be reflected also in the corresponding RDF files.

The relevant properties are:

- `dcterms:accrualPeriodicity` in [§6.6.2 Property: frequency](https://w3c.github.io/dxwg/dcat/#Property:dataset_frequency). 

  Current definition: 
  > The frequency at which dataset is published.

  New definition: 
  > The frequency at which **a** dataset is published.

- `spdx:checksumValue` in [§6.17.2 Property: checksum value](https://w3c.github.io/dxwg/dcat/#Property:checksum_checksum_value). 

  Current definition: 
  > The checksumValue property provides a lower case hexidecimal encoded digest value produced using a specific algorithm [[SPDX]].

  New definition: 
  > The checksumValue property provides a **lowercase hexadecimal** encoded digest value produced using a specific algorithm [[SPDX]]."

**Comment by riccardoAlbertoni** [comment]
I guess we want to merge the changes to the HTML  pretty soon. So I would suggest dealing with the DCAT RDF  in a new PR,  or even directly in the #1482 to avoid conflict among PRs.

> > @riccardoAlbertoni said:
> > > We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.
> > 
> > 
> > Agreed. Let's keep this on hold until #1486 is merged.
> 
> I updated the changelog, and I also took the opportunity to fix some typos and editorial issues. I also run the US spellchecker.
> 
> Some of the typos concern two property definitions not in the DCAT namespace. So the fixes should be reflected also in the corresponding RDF files.
> 
> The relevant properties are:
> 
> * `dcterms:accrualPeriodicity` in [§6.6.2 Property: frequency](https://w3c.github.io/dxwg/dcat/#Property:dataset_frequency).
>   Current definition:
>   > The frequency at which dataset is published.
>   
>   
>   New definition:
>   > The frequency at which **a** dataset is published.
> * `spdx:checksumValue` in [§6.17.2 Property: checksum value](https://w3c.github.io/dxwg/dcat/#Property:checksum_checksum_value).
>   Current definition:
>   > The checksumValue property provides a lower case hexidecimal encoded digest value produced using a specific algorithm [[SPDX]].
>   
>   
>   New definition:
>   > The checksumValue property provides a **lowercase hexadecimal** encoded digest value produced using a specific algorithm [[SPDX]]."

**Comment by andrea-perego** [comment]
@riccardoAlbertoni scripsit:

> I guess we want to merge the changes to the HTML pretty soon. So I would suggest dealing with the DCAT RDF in a new PR, or even directly in the #1482 to avoid conflict among PRs.

Yes, this was just meant to be a heads-up. And doing the fix in the RDF might not be so straightforward, as the RDF files with external properties / classes are still those of DCAT2.

**Comment by andrea-perego** [comment]
I created a PR (https://github.com/w3c/dxwg/pull/1491) on top of this one to address issue https://github.com/w3c/dxwg/issues/1490

If you think that the proposal in PR https://github.com/w3c/dxwg/pull/1491 makes sense, I suggest we wait to merge this one. Otherwise, I'll create another one.

**Review by davebrowning — APPROVED** [review]

**Pull request merged**
